### PR TITLE
Added Parse SDK package

### PR DESCRIPTION
**Pull request by espipj**
# Why

Added Parse SDK for the [Parse Platform](https://parseplatform.org/)
- I _think_ it might work on Expo, and Web as the SDK was initially focused on vanilla JS (for web) but I haven't used it on those. Can confirm that it **works on iOS and Android**.

# Checklist

If you added a new library:

- [x] Added it to **react-native-libraries.json**

### CODE REVIEW

**Comment by brentvatne** [comment]
hey there! I'm not sure about adding this one, it's not really specific to react-native. maybe we need to do some more thinking about what packages belong here

**Comment by brentvatne** [comment]
also - if it works on ios and android without needing you to install separate native modules then it works in expo client also

File: react-native-libraries.json [hunk 1]
```diff
@@ -3679,5 +3679,15 @@
     "windows": false,
     "unmaintained": false,
     "npmPkg": "react-native-voip-push-notification"
+  },
+  {
+    "githubUrl": "https://github.com/parse-community/Parse-SDK-JS",
+    "ios": true,
+    "android": true,
+    "web": false,
```
Comment (brentvatne) [inline react-native-libraries.json:3687]: if this was made originally for web should it not work on web?

Comment (brentvatne) [inline react-native-libraries.json:3687]:
```suggestion
    "web": true,
```

File: react-native-libraries.json [hunk 2]
```diff
@@ -3679,5 +3679,15 @@
...
+    "expo": false,
```
Comment (brentvatne) [inline react-native-libraries.json:3688]:
```suggestion
    "expo": true,
```

**Comment by brentvatne** [comment]
I think we need to be selective about the libraries that we add that aren't *specifically for React Native*, and include them only if 1) they are popular 2) it will help clarify any ambiguity around whether the library is compatible

**Comment by espipj** [comment]
Hey @brentvatne ! Actually you are right, I am not 100% sure if this should be included, as it is not specific package for react-native like `react-native-firebase`, altough it has an specific interface [parse/react-native](https://github.com/parse-community/Parse-SDK-JS/blob/master/react-native.js)

**Comment by espipj** [comment]
Oh, seems that I am late 😅

**Comment by espipj** [comment]
> also - if it works on ios and android without needing you to install separate native modules then it works in expo client also

Yes it does, I didn't write down Expo compatible as I've never tried it with Expo myself

**Pull request merged**